torch/csrc/inductor/aot_runtime/model.h

```diff
@@ -46,11 +59,11 @@ class AOTInductorModelBase {
         outputs_info_(num_outputs),
         constants_info_(num_constants),
         cubin_dir_(cubin_dir) {
-    C10_CUDA_CHECK(cudaEventCreate(&run_finished_));
+    ATOI_RUNTIME_CUDA_CHECK(cudaEventCreate(&run_finished_));
   }
 
-  ~AOTInductorModelBase() {
-    C10_CUDA_CHECK(cudaEventDestroy(run_finished_));
+  ~AOTInductorModelBase() noexcept(false) {
+    ATOI_RUNTIME_CUDA_CHECK(cudaEventDestroy(run_finished_));
```

Comment:
BTW, it's better to just suppress the error here if it does error, throwing in the destructor will cause an immediate terminate which will make prod teams unhappy with you

Reply:
will change